README.md

```diff
@@ -213,13 +255,16 @@ The following environment parameters are available:
  APPOPTICS_CUSTOM_TAGS          | Set this to a comma separated K=V list to enable custom tags eg. `NAME=TEST,IS_PRODUCTION=false,VERSION=5`
  SOLARWINDS_TOKEN               | Your SolarWinds token. This parameter is required.
  APPOPTICS_TOKEN                | Depreciated. Your SolarWinds token. This parameter is used as fallback if SOLARWINDS_TOKEN is not present.
+ LOGGLY_TOKEN                   | Optional. Use this when your Loggly token differs from Your SolarWinds token. If set, this will be used for tasks using Loggly Publishers.
+ PAPERTRAIL_TOKEN               | Optional. Use this when your Papertrail token differs from Your SolarWinds token. If set, this will be used for tasks using Papertrail Publishers.
  APPOPTICS_HOSTNAME             | This value overrides the hostname tagged for default host metrics. The DaemonSet uses this to override with Node name.
  LOG_LEVEL                      | Expected value: DEBUG, INFO, WARN, ERROR or FATAL. Default value is WARN.
  LOG_PATH                       | Set this value to enable SolarWinds Snap Agent logging to file. Default logs are printed to stdout for SolarWinds Snap Agent running in Docker container. Overriding this option disable reading Snap Agent log using `docker logs`, or `kubectl logs`.
  SWISNAP_SECURE                 | Set this to `true` to run only signed plugins.
  SWISNAP_DISABLE_HOSTAGENT      | Set this to `true` to disable the SolarWinds Snap Agent system metrics collection.
  SWISNAP_DISABLE_PROCESSES      | Set this to `true` to disable the SolarWinds Snap Agent processes metrics collection.
- SWISNAP_ENABLE_DOCKER          | Set this to `true` to enable the Docker plugin.
+ SWISNAP_ENABLE_DOCKER          | Set this to `true` to enable the Docker plugin. This requires Docker socket mounted inside container (done by default in DaemonSet).
+ SWISNAP_ENABLE_DOCKER_LOGS     | Set this to true to enable Logs collector task for gathering Docker logs. This requires Docker socket mounted inside container (done by default in DaemonSet).
  SWISNAP_ENABLE_APACHE          | Set this to `true` to enable the Apache plugin.
  SWISNAP_ENABLE_ELASTICSEARCH   | Set this to `true` to enable the Elasticsearch plugin.
  SWISNAP_ENABLE_KUBERNETES      | Set this to `true` to enable the Kubernetes plugin. Enabling this option on the DaemonSet will cause replication of Kubernetes metrics where the replication count is the number of pods with Kubernetes collection enabled minus one. Typically Kubernetes collection is only enabled on the Deployment asset.
```

Comment:
are we going to have a separate flag for k8s event like eg `SWISNAP_ENABLE_KUBERNETES_EVENTS` ?

Reply:
No. There is new separate instruction for providing task file as config map which is mounted to pod. 

In case of events I assume that there will be a need to modify default event filters to align to one needs. This was handled this way even with old logs suite (allow to manual modifications)

Reply:
hmmmm, good point, if there is enable_docker which enables default config, there should be enable_kube_events which does the same.

Reply:
OK, then. I will add it also.

Reply:
One additional comment:
docker logs in default config needs only to have docker socket avialalble (mounted) with Read permissions, but in case of K8S logs depending on if it's inside k8s cluster or outsite needs:

- kube config available (non incluster), 
- or in case of incluster  it need `incluster` option to set to true in task manifest and proper RBAC role added so it would have proper permissions in cluster itself.

May I ask for confirmation that we with  `SWISNAP_ENABLE_KUBERNETES_LOGS` we would like to have 1st option (default), when it's up to user to provide (mount) .k8s config to container? I'm not sure if this would be best practice tho to expect someone to mount cluster config to docker.

Reply:
well, we are incluster, so option 2 should be enabled. By "default" values I was thinking about default event filters, which would be really hard to configure using env variables.